client/state/notices/reducer.js

```diff
@@ -9,28 +9,29 @@ import { combineReducers } from 'redux';
 import {
 	NEW_NOTICE,
 	REMOVE_NOTICE,
-	SET_ROUTE
+	SET_ROUTE,
+	SERIALIZE,
+	DESERIALIZE
 } from 'state/action-types';
 
 export function items( state = [], action ) {
 	switch ( action.type ) {
 		case NEW_NOTICE:
-			state = [ action.notice, ...state ];
-			break;
+			return [ action.notice, ...state ];
```

Comment:
Personally I submitted to the guidelines in some of the stuff I've done but I think it's the ugliest (and pretty error-prone) thing. I _prefer_ allowing to return from the case statements because the noise of all those breaks just gets in the way and so often I see a huge stack of `breaks` and then at the end there's nothing more than a `return`. I'd rather see us use the control statements that best represent the action being taken.